vispy/app/base.py

```diff
@@ -184,12 +184,15 @@ def _vispy_mouse_move(self, **kwargs):
 
     def _vispy_mouse_release(self, **kwargs):
         # default method for delivering mouse release events to the canvas
+
         kwargs.update(self._vispy_mouse_data)
         ev = self._vispy_canvas.events.mouse_release(**kwargs)
-        if ev.button == self._vispy_mouse_data['press_event'].button:
+        if (self._vispy_mouse_data['press_event'] 
+                and self._vispy_mouse_data['press_event'].button == ev.button):
```

Comment:
Almost there... I'm 90% sure Travis will complain about this, too, because the binary operator `and` needs to be on the line above. That's a relatively recent PEP8 change that you'll see if you update your PEP8 checker.